spec/system.spec.js

```diff
@@ -347,21 +380,36 @@ describe('corp-semantic-release', function() {
   });
 
 
-  it('should only run post-success script when the git push command is successful', function() {
+  it('should run post-success script when the git push command is successful', function() {
     commitFeat();
-    const out = semanticRelease(`--post-success do-publish -v`);
+    const result = semanticRelease(`--post-success "npm run do-publish" -v`);
```

Comment:
I can't quite follow how this test is passing now. From what I understand, L385 will detect that a release is required and call `git push`. Does `git push` within this system test return 0 now (instead of 128)? Can you explain further, please?